lib/workerholic/cli.rb

```diff
@@ -29,7 +29,7 @@ def parse_options
       OptionParser.new do |opts|
         opts.banner = 'Usage: workerholic [options]'
 
-        opts.on '-ab', '--auto-balance', 'auto-balance workers based on number of jobs in each queue' do
+        opts.on '-a', '--auto-balance', 'auto-balance workers based on number of jobs in each queue' do
```

Comment:
The problem is that the short style switch can only contain 1 letter. That's the normal behavior for this type of argument syntax. If you check your version, it will still show 'a' (not 'ab') and append 'b' to the long style switch instead.